components/rmrk/Gallery/Holding.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <Holder
+    :events="ownerEventsOfNft"
+    group-key-option="CollectionId"
+    openOnDefault
+    date-header-label="Last Activity"
+    name-header-label="Collection"
+    :collapseTitleOption="$t('Holdings')"
+    hideCollapse />
+</template>
+
+<script lang="ts">
+import PrefixMixin from '@/utils/mixins/prefixMixin'
+import { Component, Prop, Watch, mixins } from 'nuxt-property-decorator'
+import { Interaction } from '../service/scheme'
+import allCollectionSaleEvents from '@/queries/rmrk/subsquid/allCollectionSaleEvents.graphql'
+import { notificationTypes, showNotification } from '@/utils/notification'
+import { sortedEventByDate } from '~/utils/sorting'
+import nftListOwned from '@/queries/nftListOwned.graphql'
+
+const components = {
+  Holder: () => import('@/components/rmrk/Gallery/Holder/Holder.vue'),
+}
+
+@Component({ components })
+export default class Holding extends mixins(PrefixMixin) {
+  @Prop({ type: String, default: '' }) accountId: string
```

Comment:
you can use `AuthMixin` to get `accountId`

Reply:
the accountId comes from AuthMixin is for the current user.

On the User profile page, accountId would be different for different users.